ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/422__model_serving_llm.robot

```diff
@@ -0,0 +1,204 @@
+*** Settings ***
+Documentation    Collection of tests to validate the model serving stack for Large Language Models (LLM)
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource          ../../../Resources/OCP.resource
+Resource          ../../../Resources/Page/Operators/ISVs.resource
+Suite Setup       Install Model Serving Stack Dependencies
+# Suite Teardown
+
+
+*** Variables ***
+${DEFAULT_OP_NS}=    openshift-operators
+${LLM_RESOURCES_DIRPATH}=    ods_ci/tests/Resources/Files/llm
+${SERVERLESS_OP_NAME}=     serverless-operator
+${SERVERLESS_SUB_NAME}=    serverless-operator
```

Comment:
Do we need sub namespace as another variable it will be always be same as operator namespace.

Reply:
>Do we need sub namespace


it's the subscription name, not the namespace. Initially I thought of assigning a different name for subscription in a way we can easily identity if the operator was previously installed or installed by our script (on debug time on any cluster). Wdyt?

Reply:
as of now it has the same name as the operator because I was testing on a cluster where the subscriptions where already in place (manually installed) and using a different name would have broken everything

Reply:
Ohh sorry missed it..we can have same name to be more accurate with Manual install. ..what I was actually saying is in the below keyword where you are using the variable you don't need two varible since they are having the same name we can just use single variable at both the places.

Reply:
yeah got it. As I said, I did so to give the option to have different names which i thought would have been good to have, but yeah we can change this, np